Formula/u/umka.rb

```diff
@@ -0,0 +1,32 @@
+class Umka < Formula
+  desc "Statically typed embeddable scripting language"
+  homepage "https://github.com/vtereshkov/umka-lang"
+  url "https://github.com/vtereshkov/umka-lang/archive/refs/tags/v1.5.1.tar.gz"
+  sha256 "d5f4071f142ecb8859663e41654594b9d53fddfc0f60f9e9dd0fedda9f921b99"
+  license "BSD-2-Clause"
+  head "https://github.com/vtereshkov/umka-lang.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  def install
+    system "make", "all"
+    bin.install "build/umka"
+    # include libraries
+    lib.install "build/libumka.a"
+    lib.install "build/libumka.so"
```

Comment:
actually it should be this
```suggestion
    lib.install shared_library("libumka")
```

Reply:
shouldn't it contain `build` in the path?

Reply:
yeah, it should, updated my comment.

Reply:
the dylib is not portable

Reply:
as I can see `shared_library()` method works only with `.dylib`s, not `.so`s
<img width="771" alt="image" src="https://github.com/user-attachments/assets/4e28e56c-062f-4e29-a84b-cb42f7d18bca" />

Reply:
yeah, it is wrong.

Reply:
for now I added your patch to the formula

Reply:
eh, it didn't end up well